src/debug.test.js

```diff
@@ -8,7 +8,7 @@ describe('Make sure debug-bar loads', () => {
 describe('Load data', () => {
   it('Should load data from fixtures', () => {
     cy.visit('http://localhost:3000');
-    cy.get('#debug-bar-fixture-1').click();
+    cy.get('select').select('slide: text-box top');
```

Comment:
It's better to use a class or ID as selector, since other 'select' elements might exist.